src/control_receiver.py

```diff
@@ -145,10 +149,7 @@ def message_handler(self, buf):
             # Handle configuration requests
             elif (control == "UPDATE_CONFIG"):
```

Comment:
I'd be open to renaming this to `SAVE_CONFIG` before v1.0 release. Thoughts?

Reply:
Agreed, since that control message is now handling saving, I think it's more appropriate!